mongomock/database.py

```diff
@@ -68,10 +72,37 @@ def collection_names(self, include_system_collections=True, session=None):
 
         return self.list_collection_names(session=session)
 
-    def list_collection_names(self, session=None):
+    def _verify_list_collection_supported_op(self, keys):
```

Comment:
You are not using self, so you can move this as a module functionn (closer to the set above).

Reply:
done